### PR TITLE
Feature/chart.js

### PR DESCRIPTION
**Pull request by itsnameissteven**
### Description
* Is this a feature or a fix?
A feature with some refactoring
* Where should the reviewer start?
Review the new chart.js file. Then the script.js for refactored funcitons. Minor changes made to index.html for adding charts.

One section is contains 6 charts. this will be broken down to two sections in future updates
* What functionalities do these changes effect? Why were these changes made? What was the motivation behind these changes?
User Data is now displayed utilizing charts.  
* What outside features or research did you use to implement this fix?
chart.js
* How should this be tested?
git checkout main
git fetch
npm install chart.js --save

open the web app locally.
### Relevant Screen Shots
***
#### Please review considerations below:
- [X] Follows Turing Style Guidelines
- [ ] Changes generate no new warnings or errors
- [X] Checked code for and corrected any spelling errors
- [ ] README has been updated(if applicable)

### CODE REVIEW

File: src/Hydration.js [hunk 1]
```diff
@@ -6,7 +6,12 @@ class Hydration extends Calculation {
   }
 
   calculateFirstWeekOunces(userRepo, id) {
-    return userRepo.getFirstWeek(id, this.dataSet).map((data) => `${data.date}: ${data.numOunces}`);
+    return userRepo.getFirstWeek(id, this.dataSet).map((data) => {
+      return {
```
Comment (aemiers) [inline src/Hydration.js:10]: This format looks so much cleaner!

File: src/dataChart.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                  ticks: {
+                      beginAtZero: true
+                  }
+              }]
+          }
+        }
+    });
+  },
+
+  createDoubleDataBarChart(dataSetUser, dataSetAll, element, label) {
```
Comment (aemiers) [inline src/dataChart.js:37]: Chart boss!

File: src/scripts.js [hunk 3]
```diff
@@ -17,40 +17,47 @@ import Activity from './Activity';
...
+// var userStepsThisWeek = document.getElementById('userStepsThisWeek');
+// var userStairsThisWeek = document.getElementById('userStairsThisWeek');
+// var userMinutesThisWeek = document.getElementById('userMinutesThisWeek');
+// var bestUserSteps = document.getElementById('bestUserSteps');
+// var streakList = document.getElementById('streakList');
+// var streakListMinutes = document.getElementById('streakListMinutes')
+// let hydroChart = document.getElementById("hydroChart")
+// let sleepChart = document.getElementById("sleepChart")
+
+
```
Comment (alexthompson207) [inline src/scripts.js:60]: Awesome job of moving of all these global DOM variables into their functions! Looks super clean!

File: src/scripts.js [hunk 4]
```diff
@@ -142,71 +137,63 @@ function makeWinnerID(activityInfo, user, dateString, userStorage) {
...
+  const sleepQualityToday = document.getElementById('sleepQualityToday');
+  const avUserSleepQuality = document.getElementById('avUserSleepQuality');
+  const sleepChart = document.getElementById("sleepChart");
+  sleepToday.innerText = `${sleepInfo.calculateDailyData(id, dateString, "hoursSlept")}hrs`;
+  sleepQualityToday.innerText = `${sleepInfo.calculateDailyData(id, dateString, "sleepQuality")}/5`;
+  avUserSleepQuality.innerText = `${Math.round(sleepInfo.calculateAllUserSleepQuality() * 100) / 100}/5`;
+  chart.makeChart(sleepInfo.calculateWeeklyData(dateString, id, userStorage, "hoursSlept"), sleepChart, "Hours of Sleep");
 }
 
 function addActivityInfo(id, activityInfo, dateString, userStorage, laterDateString, user, winnerId) {
```
Comment (alexthompson207) [inline src/scripts.js:170]: This is so much cleaner!

**Review by aemiers — APPROVED** [review]

**Pull request merged**